blog/admin.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib import admin
+
+# Register your models here.
+from .models import Post
+
+class PostAdmin(admin.ModelAdmin):
+    list_display = ('title', 'slug', 'status','created_on')
+    list_filter = ("status",)
```

Comment:
use consistent single and double quotes when possible. 
Personally, I prefer single quotes.